stan/math/prim/scal/prob/beta_proportion_lccdf.hpp

```diff
@@ -0,0 +1,138 @@
+#ifndef STAN_MATH_PRIM_SCAL_PROB_BETA_PROPORTION_LCCDF_HPP
+#define STAN_MATH_PRIM_SCAL_PROB_BETA_PROPORTION_LCCDF_HPP
+
+#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
+#include <stan/math/prim/scal/meta/partials_return_type.hpp>
+#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_less_or_equal.hpp>
+#include <stan/math/prim/scal/err/check_nonnegative.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_positive_finite.hpp>
+#include <stan/math/prim/scal/fun/size_zero.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/fun/digamma.hpp>
+#include <stan/math/prim/scal/fun/lbeta.hpp>
+#include <stan/math/prim/scal/meta/contains_nonconstant_struct.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <stan/math/prim/scal/meta/VectorBuilder.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <stan/math/prim/scal/fun/grad_reg_inc_beta.hpp>
+#include <stan/math/prim/scal/fun/inc_beta.hpp>
+#include <cmath>
+
+namespace stan {
+namespace math {
+
+/**
+ * Returns the beta log complementary cumulative distribution function
+ * for the given probability and parameterized with location and
+ * precision parameters. Given matching containers returns the log sum
+ * of probabilities.
+ *
+ * @tparam T_y type of probability parameter
+ * @tparam T_loc type of location parameter
+ * @tparam T_prec type of precision parameter
+ * @param y probability parameter
```

Comment:
the probablity isn't a parameter per se in this case---it's the variate whose complementary quantile is going to be returned.

Reply:
Agreed, but I've got nothing on a pithy phrase for this.  Do you have a recommendation different than `variate whose complementary quantile is returned`?

Reply:
Yes, just "type of variate" is fine, or "type of probability variate" if you want to hint at the constraint at the same time.